src/environment.go

```diff
@@ -185,7 +185,8 @@ func (env *environment) getcwd() string {
 	}
 	correctPath := func(pwd string) string {
 		// on Windows, and being case sensitive and not consistent and all, this gives silly issues
-		return strings.Replace(pwd, "c:", "C:", 1)
+		driveLetter := getCompiledRegex(`^[a-z]:`)
+		return driveLetter.ReplaceAllStringFunc(pwd, func(letter string) string {return strings.ToUpper(letter)})
```

Comment:
you can do this directly: `return driveLetter.ReplaceAllStringFunc(pwd, strings.ToUpper)`

Reply:
Much nicer, thanks. I'll make the change and push it again.

(I _did_ say I don't know Go 😄 )